<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorEditWindow</class>
 <widget class="QDialog" name="MonitorEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="monitorDetailHeaderLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Detail for Device</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="generalGroup">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="installationDateLabel">
            <property name="text">
             <string>Date Of Installation</string>
            </property>
            <property name="buddy">
             <cstring>installationDataEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="deviceCodeEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="descriptionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
            <property name="buddy">
             <cstring>descriptionEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="deviceCodeLabel">
            <property name="text">
             <string>Device code</string>
            </property>
            <property name="buddy">
             <cstring>deviceCodeEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDateTimeEdit" name="installationDataEdit"/>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QTextEdit" name="descriptionEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="statusGroup">
       <property name="title">
        <string>Status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="deviceTypeGroup">
          <property name="title">
           <string>Device Type</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="CCTVButton">
             <property name="text">
              <string>CCTV</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="cellularMonitorButton">
             <property name="text">
              <string>Cellular Monitor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="trafficMonitorButton">
             <property name="text">
              <string>Traffic Monitor</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Work Status</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="runningStateButton">
             <property name="text">
              <string>Running</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="downStateButton">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="notInstalledStateButton">
             <property name="text">
              <string>Not Installed</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
